src/api/store/page_settings.py

```diff
@@ -61,10 +61,15 @@ def update_page_setting(self, page_setting_id, args) -> (dict, MassEnergizeAPIEr
     page_setting = page_setting.first()
 
     if image:
-      media = Media.objects.create(name="Page-Image", file=image)
-      page_setting.image = media
+      # Page settings models support multiple images, but we currently allow just one.
+      if not image == "None":
```

Comment:
i think we should rather make it clearer by doing 
if image == "None" then we clear the thing else: do something else